docs/kvm/README.md

```diff
@@ -33,13 +33,16 @@ sudo ./kvm -v 3 -logtostderr
 
 ### Docker
 ```
-docker run -it -v /dev:/dev -v /sys:/sys -v /lib/modules:/lib/modules -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugins --privileged --cap-add=ALL kvm:latest /bin/bash
+docker run -it -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugins --privileged --cap-add=ALL kvm:latest /bin/bash
 (in docker image) ./kvm -v 3 -logtostderr
 ```
 
 ## As a DaemonSet
 
-NOTE: This process is not finalized yet.
+```
+kubectl apply -f manifests/kvm-ds.yml
+kubectl apply -f docs/kvm/consumer.yml
```

Comment:
This should be omitted as the consumer is just an example.

Reply:
Is it fine to note that this is an optinal line, and an example of how to consume it?

Reply:
Yup, something along that lines is fine. Let's just make sure it doesn't look like mandatory thing to run in the cluster.